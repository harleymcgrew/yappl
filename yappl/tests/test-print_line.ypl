~print_line 1;
~print_line -2.0;
~print_line false;
~print_line [104, 101, 108, 108, 111, 32, 119, 111, 114, 108, 100]
