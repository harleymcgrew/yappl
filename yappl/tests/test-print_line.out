1
-2.0
false
hello world